exercise-analyst-server/API/Services/Common/Response.cs

```diff
@@ -14,17 +14,6 @@ public class Response<T>
 
     public class Response
     {
-        public Response(HttpStatusCode httpStatusCode)
```

Comment:
Dlaczego usunęliśmy stąd konstruktory?

Reply:
imho tu  nie ma potrzeby bo  i tak wszystko idzie przez te śmieszną inicjalizacje ale to też od odwrócenia pójdzie

Reply:
A ładne były, amerykańskie takie :(